archiver.go

```diff
@@ -718,30 +752,65 @@ func CreateOrgArchives(ctx context.Context, now time.Time, config *Config, db *s
 	var archives []*Archive
 	if len(existing) == 0 {
 		// no existing archives means this might be a backfill, figure out if there are full monthes we can build first
-		archives, err = GetMissingMonthArchives(existing, now, org, archiveType)
+		archives, err = GetMissingMonthArchives(ctx, db, now, org, archiveType)
 		if err != nil {
 			log.WithError(err).Error("error calculating missing monthly archives")
 			return nil, err
 		}
 
-		// then add in daily archives taking into account the monthly that will be built
-		daily, err := GetMissingDayArchives(archives, now, org, archiveType)
+		// we first create monthly archives
```

Comment:
becuse we use DB to get missing archives, we first need to build Monthly archives and then build Daily archives